OtterLib/network/src/nothing.cpp

```diff
@@ -0,0 +1,6 @@
+/*
```

Comment:
Exlaine the use pls

Reply:
The goal of the file is to permite the ci to build because actually there is no way to compile only includes files